TEST_APPS/device/socket_app/cmd_socket.cpp

```diff
@@ -1226,49 +1193,26 @@ static int cmd_socket(int argc, char *argv[])
     }
 
     /*
-     * Commands for TCPServer
+     * Commands for TCPSocket
      * listen, accept
      */
     if (COMMAND_IS("listen")) {
         int32_t backlog;
         if (cmd_parameter_int(argc, argv, "listen", &backlog)) {
-            return handle_nsapi_error("TCPServer::listen()", info->socket().listen(backlog));
+            return handle_nsapi_error("TCPSocket::listen()", info->socket().listen(backlog));
         } else {
-            return handle_nsapi_error("TCPServer::listen()", info->socket().listen());
+            return handle_nsapi_error("TCPSocket::listen()", info->socket().listen());
         }
 
     } else if (COMMAND_IS("accept")) {
         nsapi_error_t ret;
-
-        if (info->type() != SInfo::TCP_SERVER) {
             Socket *new_sock = info->socket().accept(&ret);
```

Comment:
Intendation is now off by one, as you removed the `if() else()` clause.

Reply:
Corrected